### PR TITLE
Using ES6 syntax

### PR DESCRIPTION
**Pull request by gabyse1**
## Changes to the project
- Setup project files
- Divide js files into modules.
- Display the current date below the navigation bar.

### CODE REVIEW

File: scripts/index.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+import BookList from '../modules/book.js';
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
```
Comment (rloterh) [inline scripts/index.js:2]: - [ ] The luxon.js library is not loading properly, hence the date is not displaying. Kindly ensure to fix this issue.

Loading error:
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Gaby, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Modules successfully created
- Great UI design
- Good ReadMe
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scripts/index.js (hunk 1 above)
Comment (gabyse1) [reply scripts/index.js:2]: Hello @rloterh.
First of all, thank you for your feedback. 
I put the node_modules folder inside of *.gitignore* file. I kindly ask you Do you think I should push the *node_modules* folder to github repository?. At the moment, I am not using a *webpack* or similar tool, that's the reason I can not import the library as *from 'luxon'*. Could you give some suggestions to solve this?
Thank you in advance.

Comment (rloterh) [reply scripts/index.js:2]: @gabyse1 
No, the node module needs not to be pushed. It should only remain in the .gitignore file. 
Webpack is not really needed for this project. I'll have the same node modules and library as yours once I clone your online repository.

Please check if the app is running correctly on your local repository, if it does, then you probably need to update your code on the online repository with your local repository version. Else please try to debug your code.

Happy coding!

Comment (gabyse1) [reply scripts/index.js:2]: Hello @rloterh.
Thank you very much for your suggestion.
I tested the project and it is running correctly on my local repository. I also remove and created again package and package-lock json files. But I saw It doesn't work on the remote repository.
I don't know what other thing can I do.
Thank you.

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,359 @@
+/* --- Local styles --- */
```
Comment (D4nielJ) [inline css/index.css:1]: [OPTIONAL] The first few books that someone adds are getting a little clamp on top of each other, maybe you would like to look into that and improve upon the current styles.

image

**Review by D4nielJ — APPROVED** [review]
## Hello @gabyse1 and @zhadier, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**